ddtrace/contrib/flask/middleware.py

```diff
@@ -156,19 +155,31 @@ def _finish_span(self, span, exception=None):
         method = ""
         endpoint = ""
         url = ""
+        query = ""
+        headers = ""
+
         if request:
-            store_request_headers(request.headers, span, config.flask)
             method = request.method
             endpoint = request.endpoint or code
             url = request.base_url or ""
+            query = request.query_string.decode()
```

Comment:
We might want to safe-guard this `decode()`. Maybe something like

```suggestion
            query = request.query_string.decode() if hasattr(request.query_string, "decode") else request.query_string
```